doc/source/conf.py

```diff
@@ -15,9 +15,19 @@
 
 import datetime
 import subprocess
+import warnings
+
+warnings.filterwarnings(
+    "ignore",
+    message="DCDReader currently makes independent timesteps",
+    category=DeprecationWarning,
+    module="MDAnalysis.coordinates.DCD",
+)
 
 import MDAnalysis as mda
 
+MDAnalysis_version = mda.__version__
```

Comment:
This line should not be necessary, please remove again. (Or please explain why it was added.)